frontend/leaderboard/src/Challanges.re

```diff
@@ -0,0 +1,30 @@
+let calculateBlocksCreated = blocks => {
+  Js.log("calculateBlocksCreated");
+  Js.log(blocks);
+  Array.map((block: Types.NewBlock.t) => {Js.log(block.data)}, blocks);
+};
+
+let calculateTransactionSent = blocks => {
+  Js.log("calculateTransactionSent");
+  Array.map((block: Types.NewBlock.t) => {Js.log(block.data)}, blocks);
+};
+
+let calculateSnarkWorkCreated = blocks => {
+  Js.log("calculateSnarkWorkCreated");
+  Array.map((block: Types.NewBlock.t) => {Js.log(block.data)}, blocks);
+};
+
+let handleMetrics = (metrics, blocks) => {
```

Comment:
We might want a different datastructure than an array here but this seems ok while we work on the metrics themselves